subsystems/data_destination.go

```diff
@@ -11,6 +11,9 @@ import (
 // Do not use it.
```

Comment:
Update comment to remove use of "sink".

Reply:
Do you have a recommendation for different wording? I'm not sure what I would say differently here.

Reply:
Ryan had a good suggestion. How does that sound?